large-message-core/src/main/java/com/bakdata/kafka/AbstractLargeMessageConfig.java

```diff
@@ -282,18 +305,40 @@ private boolean enableAmazonS3PathStyleAccess() {
 
             if (!isEmpty(jwtPath)) {
                 final AWSCredentialsProvider oidcProvider = WebIdentityTokenCredentialsProvider.builder()
-                                .webIdentityTokenFile(jwtPath)
-                                .roleArn(roleArn)
-                                .roleSessionName(roleSessionName)
-                                .build();
+                        .webIdentityTokenFile(jwtPath)
+                        .roleArn(roleArn)
+                        .roleSessionName(roleSessionName)
+                        .build();
 
                 return Optional.of(oidcProvider);
             }
 
-
         }
 
         return Optional.empty();
     }
 
+    /**
+     * This method builds the Google Storage Client object. If you don't specify credentials when constructing the
+     * client, the client library will look for credentials via the environment variable GOOGLE_APPLICATION_CREDENTIALS.
+     * If the environment variable GOOGLE_APPLICATION_CREDENTIALS isn't set,  Application Default Credentials (ADC) uses
+     * the service account that is attached to the resource that is running your code. For more information see the <a
+     * href="https://cloud.google.com/docs/authentication/production#automatically">official documentation</a>
+     *
+     * @return GoogleStorageClient
+     */
+    @SneakyThrows
```

Comment:
please avoid SneakyThrows

Reply:
Okay! Any suggestions on how to handle the `IOException` here?

Reply:
try catch and throw an UncheckedIOException with a proper error message